site/en/docs/web-platform/launch-handler/index.md

```diff
@@ -27,80 +30,67 @@ The Launch Handler API lets you control how your app is launched, for example, w
-`{"route_to": "auto", "navigate_existing_client": "always"}`. The allowed values for the sub-fields
-are as follows:
+If unspecified, `launch_handler` defaults to `{"client_mode": "auto"}`. The allowed values for the
+sub-fields are:
 
-- `route_to`:
-  - `new-client`: A new browsing context is created in a web app window to load the launch's target
+- `client_mode`:
+  - `navigate-new`: A new browsing context is created in a web app window to load the launch's target
     URL.
-  - `existing-client`: An existing browsing context is used, specifically the one most recently used. How the launch is handled within that
-    browsing context depends on `navigate_existing_client`.
+  - `navigate-existing`: The most recently interacted with browsing context in a web app
+    window is navigated to the launch's target URL.
+  - `focus-existing`: The most recently interacted with browsing context in a web app window
+    is chosen to handle the launch. A new `LaunchParams` object with its `targetURL` set to the
+    launch URL will be enqueued in the document's `window.launchQueue`.
   - `auto`: The behavior is up to the user agent to decide what works best for the platform. For
     example, mobile devices only support single clients and would use `existing-client`, while
-    desktop devices support multiple windows and would use `new-client` to avoid data loss.
-
-- `navigate_existing_client`:
-  - `always`: Existing browsing contexts chosen for launch will navigate the browsing context to the
-    launch's target URL.
-  - `never`: Existing browsing contexts chosen for launch will not be navigated and instead have
-    `targetURL` in the enqueued `LaunchParams` set to the launch's target URL.
-
-{% Aside 'warning' %}
-Setting `navigate_existing_client` to `always` can cause data loss. Only use this
-when it is safe to do so (for example, in a music player), or make sure the user's data gets
-saved before navigating away from the previous location.
-{% endAside%}
+    desktop devices support multiple windows and would use `navigate-new` to avoid data loss.
 
-Both `route_to` and `navigate_existing_client` also accept a list (array) of values, where the first
-valid value will be used. This is to allow new values to be added to the spec without breaking
-backwards compatibility with existing implementations.
+The `client_mode` property also accepts a list (array) of values, where the first valid value will be
+used. This is to allow new values to be added to the spec without breaking backwards compatibility
+with existing implementations.
 
 For example, if the hypothetical value `"matching-url-client"` were added, sites would specify
-`"route_to": ["matching-url-client", "existing-client"]` to continue to control the behavior of
-older browsers that did not support `"matching-url-client"`.
+`"client_mode": ["matching-url-client", "navigate-existing"]` to continue to control the
+behavior of older browsers that did not support `"matching-url-client"`.
```

Comment:
Tiny nit: I decided to call this `focus-matching-url` in the explainer.

Reply:
Will change to that then. Thanks!

Reply:
See https://github.com/GoogleChrome/developer.chrome.com/pull/3378.